common/app/views/fragments/items/facia_cards/meta.scala.html

```diff
@@ -3,6 +3,9 @@
 @import views.support.Format
 
 <aside class="fc-item__meta js-item__meta">
+    <a href="" data-link-name="save-for-later" class="js-save-for-later-link fc-save-for-later is-hidden">
+        @fragments.inlineSvg("bookmark", "icon")
+    </a>
```

Comment:
A button (`<button>`) element would be more semantically correct. Anchor elements should only be used for linking to other pages.